docs/man/man1/oadm-ipfailover.1

```diff
@@ -90,6 +94,10 @@ value that matches the number of nodes for the given labeled selector.
 \fB\-w\fP, \fB\-\-watch\-port\fP=80
     Port to monitor or watch for resource availability.
 
+.PP
+\fB\-\-iptables\-chain\fP=""
+    When no rule to access 224.0.0.18 multicast traffic exists, one is added to the head of this chain. When the empty string is passed. iptables is not changed. Default chain is INPUT.
```

Comment:
"Default chain is INPUT."

a) That's a sentence fragment... let's at least say "The default ..."
b) Other places (e.g. --watch-port=80) put the default in the example... should we do that and just drop this sentence about the default?